Packs/HYASInsight/Integrations/HYASInsight/README.md

```diff
@@ -623,9 +511,7 @@ Returns associated IP's for the provided hash value.
 #### Human Readable Output
 
 >### HYAS HASH-IP records for md5 : 1d0a97c41afe5540edd0a8c1fb9a0f2d
->|Associated IPs|
->|---|
->| 106.187.43.98 |
+>**No entries.**
```

Comment:
Would be better to provide an example with existing Human Readable Output entries.